SalesforceCommerceCloud/pwa-kit
# [V1.5] Webpack config: no longer assumes that `config` dir exists

# Description

**With this PR**, our webpack config now no longer assumes that the `config` dir exists. 

Otherwise, **without this PR**, it is possible to break the local dev server (it would infinitely restart). I was testing the scenario when you update from v1.4 (or older) to v1.5. While adding [`noErrorOnMissing`](https://github.com/webpack-contrib/copy-webpack-plugin#noerroronmissing) does suppress the error message, it does _not_ prevent the server from continuously restarting → see the video below

https://user-images.githubusercontent.com/847300/163902537-5da9b03a-69d7-43e1-882f-e06c13a8455a.mp4


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR

To make it easier for you to test, I've generated this project with v1.4, which I then:
1. Bumped up the `pwa-kit-react-sdk` dependency to v1.5.1
3. And that's it!   

[pwa-kit-starter-project.zip](https://github.com/SalesforceCommerceCloud/pwa-kit/files/8515328/pwa-kit-starter-project.zip)

To test the PR:
1. Unzip the project above
2. Switch to the project directory
3. Run `npm install`
4. Run `npm start` to verify the always-restarting dev server
5. Now let's switch back to our PR and monorepo
6. `cd packages/pwa-kit-react-sdk` and then `npm run prod:build`
6. Copy the content of `packages/pwa-kit-react-sdk/dist/webpack/config.js`
7. Use it to replace the project's `node_modules/pwa-kit-react-sdk/webpack/config.js`
8. From within the project, re-run `npm start`
9. Verify that the dev server runs properly this time

# Todos
- [x] Bump up the patch version

# Checklists

## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
